core/amber/src/main/python/core/runnables/network_receiver.py

```diff
@@ -59,10 +63,15 @@ def data_handler(command: bytes, table: Table) -> int:
             :return: sender credits
             """
             data_header = PythonDataHeader().parse(command)
+            marker = data_header.marker
```

Comment:
change it to `data_header.payload_type` or `data_header.frame_type`?

Reply:
Renamed.